sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Change Log azure-ai-textanalytics
+# Release History
```

Comment:
the form recognizer changelog also has the header like this. do we need to change that one as well?

Reply:
Yes, I'll do that in a moment